tools/operator/scripts/validate.sh

```diff
@@ -0,0 +1,146 @@
+#!/bin/bash -e
+
+if [[ $# != 6 && $# != 9 ]]; then
+  echo "Invalid number of arguments. Usage:"
+  echo "./validate.sh <kubeconfig path> <MSPID> <artifactsLocation> <ordererOrganizations list> <numOrderer in each org> <numChannels> <peerOrganizations list> <numpeers in each org> <peerOrganizations MSPID list>"
+  exit 1
+fi
+
+export KUBECONFIG=$1
+export CORE_PEER_LOCALMSPID=$2
+ARTIFACTS_LOCATION=$3
+IFS=,
+ORDERER_ORGS=($4)
+NUM_ORDERERS=($5)
+PEER_ORGS=($7)
+NUM_PEERS=($8)
+PEER_ORGS_MSPID=($9)
+unset IFS
+NUM_CHANNELS=$6
+ORDERERS=()
+CHANNELS=("orderersystemchannel")
+PORT=30000
+FLAG=0
+if [[ "$ARTIFACTS_LOCATION" == */ ]]; then
+  ARTIFACTS_LOCATION="${ARTIFACTS_LOCATION::-1}"
+fi
+
+for (( i=0; i < $NUM_CHANNELS; ++i ))
+do
+  CHANNELS+=("testorgschannel$i")
+done
+
+ORDERER_ORG=${ORDERER_ORGS[0]}
+export ORDERER_NAME=orderer0-$ORDERER_ORG
+CURRENT_DIR=$(cd `dirname $0` && pwd)
+export FABRIC_CFG_PATH=$CURRENT_DIR/../../../fabric/.build/config/
+
+
+exportOrdererConfig(){
```

Comment:
rename.  It is not just for orderers. this procedure does much more than export things.
If you want to also create another procedure with this name just to do the exports, that is ok.

Reply:
Could you please suggest me one Scott

Reply:
Maybe networkInSync as the function and fileName?

Reply:
how about: validateNetworkInSync

Reply:
okay, Scott. We will use this.